### PR TITLE
Add JP tenant permission

### PR DESCRIPTION
**Pull request by marounkoussaifi**

### CODE REVIEW

File: scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/UserIdentification.java [hunk 1]
```diff
@@ -91,6 +91,11 @@ public abstract class UserIdentification implements Serializable, Comparable<Use
      */
     public abstract boolean isAllTenantPermission();
 
+    /**
+     *
```
Comment (mklkun) [inline scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/UserIdentification.java:95]: Any reason for this empty description?

Comment (marounkoussaifi) [reply scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/UserIdentification.java:95]: I must have forgotten to add it. Thanks.

**Review by fviale — APPROVED** [review]
LGTM

**Comment by fviale** [comment]
Retest this please

**Pull request merged**